fileio/private/read_nervus_data.m

```diff
@@ -145,7 +146,17 @@
         % Final Partial Segment
         curSec = nrvHdr.MainIndex(useSections(end));
         fseek(h, curSec.offset,'bof');
-        out(curIdx : end,i) = fread(h, length(out)-curIdx + 1, 'int16') * mult;
+        %Sometimes there is less data available in the file.
```

Comment:
just a general comment (this is just nitpicking): the fieldtrip coding guidelines recommend to have '2 spaces' per tab. Perhaps that can be easily imposed in this file, too. Also, there's a small typo in the help section, which mentions 'GETDATA', rather than READ_NERVUS_DATA

Reply:
Done